src/Juvix/Syntax/Abstract/Language.hs

```diff
@@ -219,3 +219,11 @@ instance HasLoc AxiomDef where
 
 instance HasLoc FunctionDef where
   getLoc = getLoc . (^. funDefName)
+
+getIdenName :: Iden -> Name
```

Comment:
we already have `idenName` in `Abstract.Language.Extra`

Reply:
Thanks! I removed the duplicate function.